.eslintrc

```diff
@@ -6,6 +6,7 @@
     "key-spacing": "off",
     "guard-for-in": "off",
     "no-restricted-syntax": "off",
-    "import/no-dynamic-require": "off"
+    "import/no-dynamic-require": "off",
+    "linebreak-style": "off"
```

Comment:
Hi @dnalborczyk, can you please explain why you are disabling that rule in that PR ? Thanks

Reply:
You can temporarily change this value in your IDE

Reply:
the windows build is failing: https://travis-ci.org/dherault/serverless-offline/builds/525380343?utm_source=github_status&utm_medium=notification  I think usually git takes care of the line break conversion itself on the host system (if not specified differently). I assume that the Travis Windows image installs git with a different setting, and changed line breaks on Windows.

found this issue: https://travis-ci.community/t/files-in-checkout-have-eol-changed-from-lf-to-crlf/349/2